.circleci/config.yml

```diff
@@ -1,4 +1,48 @@
-version: 2
+version: 2.1
+orbs:
```

Comment:
How does this orb get registered for us to use elsewhere?

Reply:
We can push it to Circle's orb registry; a lot of detail @ https://circleci.com/docs/2.0/creating-orbs/ that I haven't had a chance to dig into yet.

Reply:
Ah ok, thanks.  I figured there had to be more to this in the form of something we're not doing yet or some real dark magic I wasn't seeing.